src/main/java/kitchenpos/menus/tobe/domain/MenuProducts.java

```diff
@@ -0,0 +1,67 @@
+package kitchenpos.menus.tobe.domain;
+
+import static kitchenpos.global.utils.CollectionUtils.isEmpty;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import kitchenpos.global.vo.Price;
+import kitchenpos.global.vo.ValueObject;
+
+@Embeddable
+public class MenuProducts extends ValueObject {
+
+    @OneToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @JoinColumn(
+            name = "menu_id",
+            nullable = false,
+            columnDefinition = "binary(16)",
+            foreignKey = @ForeignKey(name = "fk_menu_product_to_menu")
+    )
+    private List<MenuProduct> values;
```

Comment:
지연초기화의 단점을 방지하기 위해 객체를 미리 만들어 놓는것은 어떤가요?
```suggestion
    private List<MenuProduct> values = new ArrayList<>();
```